app/src/main/java/io/mercury/coroutinesandbox/view/downloader/DownloadListActivity.kt

```diff
@@ -104,28 +103,22 @@ class DownloadListActivity : ComponentActivity() {
                     }
                 }
 
+                val downloaderState = remember { downloaderFeature.state }.collectAsState()
```

Comment:
Also, do we need "remember" here? 

`collectAsState` ends up using a `LaunchedEffect` which, I believe, *should* be sufficient in this case.

Reply:
Let me revisit that. We may not.